### PR TITLE
Frontend 

### PR DESCRIPTION
**Pull request by arun-hel**
- Created page layout
- Added antd
- Added 404 page
- Added config file

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
   "private": true,
   "dependencies": {
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "antd": "^5.0.5",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^6.4.5",
     "react-scripts": "5.0.1",
```
Comment (manishkatyan) [inline frontend/package.json]: put ^ before version for backward compatibility

File: frontend/public/index.html [hunk 2]
```diff
@@ -5,10 +5,7 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
+    <meta name="description" content="Class Notes" />
```
Comment (manishkatyan) [inline frontend/public/index.html]: update description to: Class Notes transcribes online classes with subtitles, summaries, topics and sentiments, enabling students, facing poor internet or language barrier, to learn better, at their own pace

**Review by manishkatyan — CHANGES_REQUESTED** [review]

File: frontend/package.json (hunk 1 above)
Comment (arun-hel) [reply frontend/package.json]: Done

File: frontend/public/index.html (hunk 2 above)
Comment (arun-hel) [reply frontend/public/index.html]: Done

**Review by nishekh-e-r — COMMENTED** [review]
looks good

**Pull request merged**